test/unit/writer/proposal.test.ts

```diff
@@ -425,11 +425,11 @@ describe('writer/proposal', () => {
           expect(mockGetSpace).toHaveBeenCalledTimes(1);
         });
 
-        it('rejects a proposal with no privacy', async () => {
+        it('accepts a proposal with empty string in privacy', async () => {
```

Comment:
This test is nested inside `'when the space is using SHUTTER privacy'`, and should accept only `shutter` as value, and not `""`

Reply:
The test above is for another scenario.

This current test is for when the space has set `SHUTTER` as privacy, and we're sending an empty string as proposal privacy.

In this case, the writer should reject, since empty string (== no privacy), and we can not disable privacy is space has enforced it.